src/main/drivers/accgyro.h

```diff
@@ -23,6 +23,7 @@ typedef struct gyro_s {
     sensorInitFuncPtr init;                                 // initialize function
     sensorReadFuncPtr read;                                 // read 3 axis data function
     sensorReadFuncPtr temperature;                          // read temperature if available
+    sensorInterruptFuncPtr intStatus;
```

Comment:
I agree. Yes I went for coding style like the others. I will leave it as it is to fit the code.